status/statusPage.html

```diff
@@ -0,0 +1,699 @@
+            title += "String: " + object['string'] + "<br>";
+            title += "Hash: " + object['hash'] + "<br>";
+            title += "Type: " + object['type'] + " <br>";
+            title += "Current_time: " + object['current_time'];
+            return title;
+        }
+
+        //Runs through the JSON file, at each object, it creates an ID based on the linux name and parameters. It searches to see if the ID exists,
+        //if it does, it assigns the box with the ID the correct color and hovertext.
+        function determineColor() {
+            readTextFile(JSONPath, function (json) {
+                let parsedJSON = JSON.parse(json);
+                for (let i in parsedJSON) {
+                    let idToTest = parsedJSON[i]['linux_name'] + " " + parsedJSON[i]['params'][0] + " " + parsedJSON[i]['params'][1] + " " + parsedJSON[i]['params'][2] + " " + parsedJSON[i]['params'][3];
+                    if (document.getElementById(idToTest)) {
+                        document.getElementById(idToTest).onmouseover = function () {
+                            document.getElementsByClassName("detailsBarText")[0].innerHTML = getDetails(parsedJSON[i]);
+                        };
+                        if (parsedJSON[i]['status'] === "Finished") {
+                            document.getElementById(idToTest).style.backgroundColor = "forestgreen";
+                        } else if (parsedJSON[i]['status'] === "Failed" && parsedJSON[i]['kill_reason'] === "timeout") {
+                            document.getElementById(idToTest).style.backgroundColor = "darkorange";
+                        } else if (parsedJSON[i]['status'] === "Failed" && parsedJSON[i]['kill_reason'] !== "timeout") {
+                            document.getElementById(idToTest).style.backgroundColor = "red";
+                        }
+                    }
+                }
+            });
+        }
+        determineColor();
+    </script>
+
+    <link href="https://fonts.googleapis.com/css?family=Righteous|Barlow+Condensed|Open+Sans" rel="stylesheet">
+</head>
+<body>
+<header>
+    <img src="https://avatars2.githubusercontent.com/u/32246645?s=280&v=4" alt="DArchR">
+    <h1>UC Davis Computer Architecture Research Group <i>Status Page</i></h1>
+</header>
+<main>
+    <!--- If you want to make a new test, create a div with an ID equal to: "linux-type test-type mem-sys cpu-number boot-type
+    You might notice there area lot of empty div tags, this is to make the grid objects work -->
+    <h2>Mem Sys: classic, Boot Type: init</h2>
+    <div class=box-of-test-grids>
+        <div class="color-code"><span style="color: red;">Failure </span><span
+                style="color: darkorange;">Timeout </span><span style="color: forestgreen;">Success</span></div>
+        <div class="test-grid">
+            <div></div>
+            <div class="test-grid-title">linux-4.4.186</div>
+            <div class="cpu-number-label">8 CPU -</div>
```

Comment:
Can we change this class to "y-axis-label"? It would make it more general, assuming this idea makes sense at all.